mesteriApplication/src/main/java/com/intern/alexx/repository/ContactRepository.java

```diff
@@ -11,9 +11,9 @@
  */
 public interface ContactRepository {
 	
-	void insert(Contact contact);
+	void insert(Contact contact, String mesterId);
```

Comment:
Contact deja are mesterId. Nu trebuie sa il trasmiti separat. Semnatura metodei este insert(Contact) .
In general repository primeste doar entitatea ca si parametru.

BaseRepo<K, T>
void insert(T)
void update(T)
void delete(K)
T findById(K)
List<T> findAll(page, pageSize)